azure/connect.sh

```diff
@@ -21,6 +21,4 @@ ssh_port=$(az deployment group show -g ${RESOURCE_GROUP_NAME} -n azuredeploy --q
 public_ip=$(echo ${public_ip} | cut -d "\"" -f 2)
 ssh_port=$(echo ${ssh_port} | cut -d "\"" -f 2)
 
-./add-local-ip.sh
```

Comment:
why do we remove that ?

Reply:
I think was some old configuration needed to allow your public ip to be connect to the VM-s. That is no longer necessary (and unreliable from my experience running the tests) since configuring and connection to the VPN should work just fine. 

This is the case for HammerDb tests. I assume this extends to the valgrind tests run on Azure VM-s as well. To make sure, it may be worth testing this.